### PR TITLE
STOR-1574: Bump library-go

### PR DESCRIPTION
**Pull request by jsafrane**
Update openshift/library-go and openshift/api to today's master. Update k8s deps to v0.29.0.
@openshift/storage

### CODE REVIEW

**Comment by jsafrane** [comment]
/label px-approved
/label docs-approved
No changes from user perspective
/label qe-approved
CI should be enough for library bumps

File: pkg/operator/operatorclient.go [hunk 1]
```diff
@@ -84,3 +84,12 @@ func (c OperatorClient) GetOperatorInstance() (*operatorv1.Storage, error) {
...
 }
+
+func (c OperatorClient) GetOperatorStateWithQuorum(ctx context.Context) (*operatorv1.OperatorSpec, *operatorv1.OperatorStatus, string, error) {
+	instance, err := c.Client.Storages().Get(ctx, globalConfigName, metav1.GetOptions{})
+	if err != nil {
+		return nil, nil, "", err
+	}
+
+	return &instance.Spec.OperatorSpec, &instance.Status.OperatorStatus, instance.GetResourceVersion(), nil
+}
```
Comment (dobsonj) [inline pkg/operator/operatorclient.go:95]: What is this for? I can't find where it's called, maybe I'm missing something.

Comment (dobsonj) [reply pkg/operator/operatorclient.go:95]: Ah nevermind, [now I see it](https://github.com/openshift/vsphere-problem-detector/pull/152/files#diff-7aa77206e39ed184ec2081444dc5add93fe4b0ae88c8bce18303a394e4956244R175)

**Pull request merged**